src/Client.php

```diff
@@ -243,6 +243,14 @@ protected function getExceptionMap()
     }
 
     /**
+     * @return LoggerInterface
+     */
+    public function getLogger()
```

Comment:
👍 

Though I disagree that this would be any help for #162 - the default `NullLogger` won't do you any good.